lib/Schedule/LongSteps.pm

```diff
@@ -476,6 +484,50 @@ sub load_process {
     return $self->_load_stored_process( $stored_process, $context );
 }
 
+sub revive {
+    my ( $self, $process_id, $revive_to ) = @_;
+
+    # find process
+    my $stored_process = $self->find_process($process_id);
+    confess "There is no $process_id to revive" unless $stored_process;
+
+    # check to see if its allowed to be revived.
+    # has it stopped
+    confess("$process_id does not have a status of 'terminated'") if ( $stored_process->status() ne "terminated" );
+
+    # load the process and check if process have the method to revive_to
+    # if revive $revive_to was not passed, used the function we failed on.
+    # and check that also, just in case we attempt to revive on a method
+    # that was previously removed.
+    my $loaded_process = $self->_load_stored_process($stored_process);
+
+    $revive_to = $stored_process->what() unless $revive_to;
+    unless ( $loaded_process->can($revive_to) ) {
```

Comment:
separate those two things. The spacing implies they are logically linked together. Which is not the case. Confuses me.